src/resources/parser/glb-parser.js

```diff
@@ -1172,12 +1180,12 @@ Object.assign(pc, function () {
     };
 
     // create engine resources from the downloaded GLB data
-    var createResources = function (device, gltf, buffers, textures, callback) {
+    var createResources = function (device, gltf, buffers, textures, options, callback) {
```

Comment:
Could we also include a `processGlobalExtras` option as some glb assets contain extras data inside the gltf object itself?

Reply:
Consider it done!